assets/dev/scss/editor/panel/_panel.scss

```diff
@@ -39,10 +39,28 @@
 		width: $panel-xl-width;
 	}
 
-	.ps-scrollbar-y-rail {
+	.ps__rail-y {
 		z-index: 2; // Avoid panel elements (like divider control) from appearing above the scrollbar
 	}
 
+	#elementor-panel-elements,
+	#elementor-global-templates,
```

Comment:
`#elementor-global-templates` belongs to the Pro. You should change the rules there

Reply:
There is no reason to duplicate the code. it should be managed from one place.

Reply:
@StyleShit can you replace it with a general class that can be used on every panel?

Reply:
Great idea!